engine/robot_state.py

```diff
@@ -0,0 +1,138 @@
+import math
+import numpy as np
+from engine.phase import Phase
+
+# Sensor dependent packages
+from engine.is_raspberrypi import is_raspberrypi
+if is_raspberrypi():
+    from electrical.motor_controller import MotorController
+    import electrical.gps as GPS 
+    import electrical.imu as IMU 
+    import electrical.radio_module as RadioModule
+    import serial
+    # from engine.sensor_module import SensorModule # IMU + GPS testing
+
+class Robot_State:
+    """
+    A class containing robot-specific information about the current state of a robot.
```

Comment:
Overall comment: The order of all of these attributes in the comment should match the order of all the attributes when defined

Reply:
good point